Documentation customchannels (#13840)
conda/conda: docs/source/user-guide/configuration/settings.rst

    - http://some.custom/channel
    - file:///some/local/directory

.. _custom_mutlichannels:

> ``custom_multichannels``: Custom channels

Reviewer: ```suggestion
``custom_multichannels``: Define custom multi-channels
```

-----------------------------------------

A multichannel is a metachannel composed of multiple channels. The two reserved multichannels are 
'defaults' and 'local'. The 'defaults' multichannel is customized using the ``default_channels``
parameter. The 'local' multichannel is a list of file:// channel locations where conda-build 
stashes built packages. 

Other multichannels can be defined with ``custom_multichannels`` in your conda configuration file
``.condarc`` using a dictionary format where keys are names of multichannels and the value is a list

> containing channel names or URLs. Channels specified here will have lower priority than default channels. 

Reviewer: ```suggestion
containing channel names or URLs. ``custom_multichannels`` configuration allows one to reference these custom multi-channels by name in ``conda`` commands or to include them in the ``channels`` list.
```
I don't believe this is true about custom multi-channels having a lower priority. As I understand a user can create a custom multi-channel and put it in the list of channels before the defaults? But I might be wrong @travishathaway

Reviewer 2: @dashagurova,

Yes, that's corrrect. A channel's priority depends a number of things and there's no reason that a multi-channel would have a lower priority than the default channel.

`defaults` (the default channel) is itself a multi-channel so that's another reason that doesn't make sense.

**Example:**

.. code-block:: yaml 
  

>   custom_multichannels:
>     my_channel: https://mychannel.example.com
>     another_channel: file:///path/to/local/channel

Reviewer: ```suggestion
  custom_multichannels:
    my_channel:
      - https://channel.example.com
      - https://anotherchannel.example.com
```

> Note: This is not a substitute for mirrors

Reviewer: ```suggestion
.. note::
  ``custom_multichannels`` is not a substitute for mirrors.
```
@travishathaway maybe you can expand on this, I am not sure how to explain why it is not a sub for mirrors.

Reviewer 2: I think it's actually best for us to leave it out entirely. I know it says to include this in the issue.

I say that because I think that will just confuse users. I will update the original issue to reflect this.

> Note: This is not a substitute for mirrors

Reviewer 2: ```suggestion
```
